app/main/views/root.py

```diff
@@ -77,15 +77,13 @@ def redirect_to_location(current_location):
     """
 
```

Comment:
Not sure what the purpose for this if block is, aren't all pages effectively going to /questionnaire/' + current_location?